packages/@aws-cdk/aws-rds/README.md

```diff
@@ -340,6 +340,7 @@ declare const instance: rds.DatabaseInstance;
 instance.addRotationSingleUser({
   automaticallyAfter: cdk.Duration.days(7), // defaults to 30 days
   excludeCharacters: '!@#$%^&*', // defaults to the set " %+~`#$&*()|[]{}:;<>?!'/@\"\\"
+  securityGroup: mySecurityGroup, // optional, defaults to an auto-created security group
```

Comment:
please use `declare` to explain what `mySecurityGroup` is
(like `instance` above this)

Reply:
Thank you for the comment. Fixed!